archstrike-testing/teamsql/teamsql.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
remove this file, add script into pkgbuild

Reply:
Fixed